xml/storage_filesystems.xml

```diff
@@ -623,78 +638,79 @@ Id Path      single   single    single   Unallocated
       In case you want to nullify an existing quota, set a quota size of
       <literal>none</literal>:
      </para>
- <screen>&prompt.sudo;btrfs qgroup limit none /var/tmp</screen>
+<screen>&prompt.sudo;btrfs qgroup limit none /var/tmp</screen>
      <para>
       To disable quota support for a partition and all its subvolumes, use
       <command>btrfs quota disable</command>:
      </para>
- <screen>&prompt.sudo;btrfs quota disable /</screen>
+<screen>&prompt.sudo;btrfs quota disable /</screen>
     </tip>
    </sect3>
    <sect3 xml:id="setting-btrfs-quotas-for-more-info">
     <title>More information</title>
     <para>
      See the <command>man 8 btrfs-qgroup</command> and <command>man 8
-      btrfs-quota</command> for more details. The
-     <citetitle>UseCases</citetitle> page on the Btrfs wiki (<link
-      xlink:href="https://btrfs.wiki.kernel.org/index.php/UseCases"/>) also
-      provides more information.
+     btrfs-quota</command> for more details. The
+     <citetitle>UseCases</citetitle> page on the Btrfs wiki
+     (<link
+      xlink:href="https://btrfs.wiki.kernel.org/index.php/UseCases"/>)
+     also provides more information.
     </para>
    </sect3>
   </sect2>
 
   <sect2 xml:id="sec-filesystems-major-btrfs-swapping">
-	  <title>Swapping on Btrfs</title>
-    <important>
-     <title>Snapshots with swapping</title>
+   <title>Swapping on Btrfs</title>
+   <important>
+    <title>Snapshots with swapping</title>
+    <para>
+     You will not be able to create a snapshot if there is any enabled swap
+     file in the source subvolume.
```

Comment:
```suggestion
     You will not be able to create a snapshot if the source subvolume contains
     any enabled swap files.
```

Reply:
The phrasing sounded a little odd to me, but I am open to further suggestions!

Reply:
I like your suggestion.